bin-src/lib/getDependentStoryFiles.js

```diff
@@ -99,7 +99,9 @@ export async function getDependentStoryFiles(ctx, stats, statsPath, changedFiles
   ctx.turboSnap.modules = Object.keys(idsByName);
 
   if (ctx.turboSnap.globs.length === 0) {
-    ctx.log.error(noCSFGlobs({ statsPath, storybookDir, viewLayer }));
+    const entry = stats.modules.find((m) => m.name.endsWith('generated-stories-entry.js'));
```

Comment:
This isn't the name of the file in the v7 store. It is called `storybook-stories.js` there: https://github.com/chromaui/chromatic-cli/blob/bf0999da3906fc50f269e6b7573103990c90c765/bin-src/lib/getDependentStoryFiles.js#L70

Reply:
Do we need to to support both?